tests/test-client-handles-client-closes-connection-unix.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
```

Comment:
should we have at least one test with an explicit `quiet` flag?

Reply:
I can do that, though we don't have good support for testing the log output atm.

Reply:
cool, at least we'll have end-to-end test that hits new code paths

Reply:
Added this now -- integration tests capture logs from stderr and check that there's no logs. I added tests for --quiet=all and --quiet=conns for now